addons/sourcemod/scripting/BossHP.sp

```diff
@@ -315,6 +315,9 @@ stock void LoadOldConfig()
 			continue;
 		}
 
+		// Prevent error and bad display
+		ReplaceString(sName, sizeof(sName), "%", "");
```

Comment:
if you need to escape some characters there is this function but idk the use case for this one:
- https://sourcemod.dev/#/dbi/function.SQL_EscapeString